examples/tutorials/colabs/rigid_object_tutorial.ipynb

```diff
@@ -630,6 +624,10 @@
  ],
  "metadata": {
   "accelerator": "GPU",
+  "colab": {
+   "name": "rigid_object_tutorial.ipynb",
```

Comment:
FYI, This is what the displayed name of the Colab will be. It can be separate from the filename.